odk/collect/collect_app/src/main/java/org/odk/collect/android/events/FormEventExporter.kt

```diff
@@ -0,0 +1,64 @@
+package org.odk.collect.android.events
+
+/**
+ * Events exporter class for form events. This class contains all the
+ * events that occur during the lifecycle of a form.
+ */
+object FormEventExporter: ODKEventExporter<FormStateEvent>() {
```

Comment:
Exporter does not seem like the right word here. Can we call them EventBus? Or EventPipeline?